README.md

```diff
@@ -34,9 +36,3 @@ Here is the planned architecture and [technologies used](https://github.com/Sene
 ![components](img/Starchart%20architecture.drawio.png)
 
 ## Development
-
-- Further technical background, planning, and initial designs are available in the [wiki](https://github.com/Seneca-CDOT/starchart/wiki).
-
-- If you would like to contribute, please see how to [here](CONTRIBUTING.md).
-
-- You can find the license [here](LICENSE).
```

Comment:
Why are we removing these?

Reply:
I've moved them up to the top, so you get to that info faster.